contracts/security/Pausable.sol

```diff
@@ -15,6 +15,16 @@ import "../utils/Context.sol";
  * simply including this module, only once the modifiers are put in place.
  */
 abstract contract Pausable is Context {
+    /**
+     * @dev The contract is paused.
+     */
+    error PausablePaused();
+
+    /**
+     * @dev The contract is not paused.
+     */
+    error PausableUnpaused();
+
```

Comment:
Paused() and Unpaused() are common error that we see here, but that will trigger in other contract (ERC20Pausable?) when they use the modifier.

IMO these error shouldn't have a domain, and should be use "as is" (either through the modifier, or manually)